### ISSUE TITLE
Align property of item doesn't override timeline align property

### ISSUE THREAD

**Issue by vankejan** [visjs/vis-timeline#29]
Hey, i have encountered a problem with `align` property of an item not overriding the `options` `align` property of timeline.

I want some specific items on the timeline to be aligned on the left of the line and some on the right side. 

Due to the [documentation](https://visjs.github.io/vis-timeline/docs/timeline/#items) for Vis.js timeline, property `align` of item should override the global `align` property of options for timeline. But it does not. 

item example:
```javascript
    var items = [];

    var item = {
      align: "left"
      date: "20.09.2019"
      eventId: 440
      heading: "Event"
      id: "86ad4f21-6235-46e0-b5c9-29a6d2c16f50"
      nodeType: "single-node"
      start: "2019-09-20T04:00:00"
    }

    items.push(item);

    // create dataset
    var dataset = new vis.DataSet(items);
```

timeline options:
```javascript
    var options = {
      orientation: 'top',
      min: "2019-09-20T04:00:00",
      max: "2019-09-25T04:00:00",
      start: "2019-09-20T04:00:00",
      end: "2019-09-25T04:00:00",
      align: 'left'
    }
```
here is how i update the item align property after calling the `vis.Timeline` constructor:
```javascript
    var visualization = document.getElementById('visualization');

    var timeline = new vis.Timeline(visualization, dataset, options);

    timeline.itemsData.forEach(function(dataItem) {
      if (dataItem.eventId === 440) {
        dataItem.align = 'right';
        timeline.itemsData.update(dataItem, {
          fieldId: "eventId"
        })
      }
    })
```
If i just use it wrongly, then i am sorry for spamming here, but i spent lots of time trying to figure it out.

**Comment by mojoaxel** [comment]
I created a minimal working example:
https://jsbin.com/viqohid/edit?js,console,output

The [item documentation](https://visjs.github.io/vis-timeline/docs/timeline/#itemOptionTable) states:
> If set this overrides the global align configuration option for this item.

I can confirm this is a bug!

**Comment by mojoaxel** [comment]
This bug exists also in vis@4.21-0-EOL and way back to vis@3.5.0 and propably earlier.

Since it looks like this never worked as described I think the easiest way is to change the docs to say: 
"The items align option does NOT overwrite the global align option".

**Comment by davidgri** [comment]
If no global align option ist set the item align isn't working either. 

[https://jsbin.com/viqohid/2/edit?js,console,output](https://jsbin.com/viqohid/2/edit?js,console,output)

**Comment by yotamberk** [comment]
Will be fixed with #205
